www/push.js

```diff
@@ -77,6 +81,9 @@ var PushNotification = function () {
 
     // triggered on error
     var fail = function fail(msg) {
+      if (msg === 'disallowed') {
```

Comment:
Probably just get rid of this and let the regular error handler take care of it.